pandas/core/generic.py

```diff
@@ -4685,6 +4685,18 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
             Whether to raise on invalid data types (e.g. trying to where on
             strings)
 
+        Examples
+        ________
+
+        >>> df = pd.DataFrame(np.arange(10).reshape(-1, 2), columns=['A', 'B'])
+        >>> df.where(df % 3, -df)
```

Comment:
as ``cond`` should be a boolean ``DataFrame``, ``df % 3 != 0`` is better. Also, I prefer ``arange(1, 11)`` as input because ``-0 = 0``